staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -104,6 +104,16 @@ const (
 	kmsReloadHealthCheckName = "kms-providers"
 )
 
+var (
+	configScheme = runtime.NewScheme()
+	codecs       = serializer.NewCodecFactory(configScheme)
+)
+
+func init() {
+	utilruntime.Must(apiserverconfig.AddToScheme(configScheme))
+	utilruntime.Must(apiserverconfigv1.AddToScheme(configScheme))
+}
```

Comment:
I think we only need the `codecs` global?

```suggestion
var codecs serializer.CodecFactory

func init() {
	configScheme := runtime.NewScheme()
	utilruntime.Must(apiserverconfig.AddToScheme(configScheme))
	utilruntime.Must(apiserverconfigv1.AddToScheme(configScheme))
	codecs = serializer.NewCodecFactory(configScheme)
}
```

Reply:
that's correct!